service/tablet_allocator.hh

```diff
@@ -124,6 +125,26 @@ public:
     void on_leadership_lost();
 };
 
+// Represents metric for per-node load which we want to equalize between nodes.
+// It's an average per-shard load in terms of tablet count.
+using load_type = double;
+
+enum class tablet_replica_calculation_status {
+    success,
+    no_tablets_to_allocate,
+    not_enough_nodes,
```

Comment:
I don't see where `no_tablets_to_allocate` or `not_enough_nodes` or `more_than_one_error` are ever set

Reply:
The whole enum has been removed.